src/UrlHandlers/CrudUrlHandler.php

```diff
@@ -29,15 +29,19 @@ class CrudUrlHandler extends LeafRestUrlHandler
 
     private $leafClassStub;
 
-    public function __construct($modelName, $namespaceBase, $additionalPresenterClassNameMap = [], $childUrlHandlers = [])
+    public function __construct($modelName, $namespaceBase, $additionalPresenterClassNameMap = [], $childUrlHandlers = [], $prefix)
```

Comment:
Needs to have = null to make it optional (can't have non optional args after options ones anyway otherwise the optional ones aren't well, optional -follow?)